expected/init_test.out.in

```diff
@@ -33,3 +33,4 @@ psql:pg_sphere.test.sql:8622: NOTICE:  argument type pointkey is only a shell
 psql:pg_sphere.test.sql:8628: NOTICE:  argument type pointkey is only a shell
 psql:pg_sphere.test.sql:8634: NOTICE:  argument type pointkey is only a shell
 psql:pg_sphere.test.sql:8640: NOTICE:  argument type pointkey is only a shell
+
```

Comment:
Why did you add this blank line? I think it should be removed.

Take a look at the diff output at the bottom of the log here:
https://app.travis-ci.com/github/postgrespro/pgsphere/jobs/607714835

You need to change the line numbers in this file to match the results so that there are no differences.